app/Repositories/CacheUserRepository.php

```diff
@@ -197,11 +233,23 @@ protected function retrieve($key)
      * Items not found in the cache will have a null value.
      *
      * @param  array  $keys
-     * @return array
+     * @return array|null
      */
     protected function retrieveMany(array $keys)
     {
-        return Cache::many($keys);
+        $retrieved = [];
+
+        $data = Cache::many($keys);
+
+        foreach ($data as $item) {
+            if ($item) {
+                $retrieved[] = $item;
+            }
+        }
+
+        if (count($retrieved)) {
+            return $data;
```

Comment:
what happens when this doesnt return anything? I feel like the return should just be outside of IF regardless of the count, no harm in returning an empty array